### ISSUE TITLE
Stop handling SQLite too many variables error

### ISSUE THREAD

**Issue by ErisDS** [TryGhost/Ghost#12032]
## Problem

People using SQLite3 in production sometimes fall foul of SQLite's 999 variable limit. We've been thinking about how to refactor to cope with this, and we have a few bits and pieces in the codebase, but actually SQLite fixed it in their 3.32 version: https://www.sqlite.org/releaselog/3_32_0.html

Hopefully Node's SQLite project will come bundled with 3.32 by default soon as it's already in master.

**Appetite:** 0.5 day

## Solution:

1. We should remove all workarounds that we have for this in the codebase, they are code we don't need to maintain anymore.
E.g. https://github.com/TryGhost/Ghost/commit/08f54d2f57f35d5c20bdd824ffef6eaac9bb9fe9
E.g. https://github.com/TryGhost/Ghost/commit/c7012935144db53d3c07e3079167de5278631c95
E.g. https://github.com/TryGhost/Ghost/commit/a29ac2691a1ae79e93ca65f46127e9fc378881c2
2. We should _maybe_ look at adding an error handler that catches the error and converts it to a nicely formatted Ghost error telling people to upgrade to SQLite 3.32 - but only if this can be done quickly and easily in one place.
> SQLITE_ERROR: too many SQL variables
3. Maybe mention this as a minimum requirement in our docs

**Comment by naz** [comment]
> Hopefully Node's SQLite project will come bundled with 3.32 by default soon as it's already in master.

@ErisDS saw [your conversation](https://github.com/mapbox/node-sqlite3/issues/1292#issuecomment-668494366) in the thread asking for maintenance help for node-sqllte3 and having commit rights to the repo. The [5.0.1 version was merged](https://github.com/mapbox/node-sqlite3/pull/1366) on GH but never published to npm because the build failed. Would be cool if you could influence a proper release of 5.0.1 (also happy to help in any way possible myself). 

To do a proper cleanup we'd need new binaries, so that Ghost could use worker_threads instead of [current workaround](https://github.com/TryGhost/Ghost/pull/12431) with child processes.

**Comment by naz** [comment]
Tried out fresh 5.0.1 build of node-sqlite3. Unfortunately it does not compile the binaries locally (my environment is Ubuntu 20.04, with Node v12, and python 3). The issue is this one described in [node-sqlite3 here](https://github.com/mapbox/node-sqlite3/issues/1415) (outdated node-gyp dependency).

Another issue I've spotted was that most popular Linux server distro hosting Ghost (Ubuntu) installs sqlite3 at version 3.31 using `apt install sqlite3`. Getting `3.32` running locally requires manual compilation of binaries.

Given above issues, I don't think it would make sense to remove the workarounds just yet. Otherwise we leave self-hosters in a world of pain trying to install latest sqlite3 version (https://github.com/TryGhost/Ghost-CLI/issues/1373 with regular sqlite3 installations already).

I think to move forward with this issue we need to have an easy way to run sqlite3 either through bundled binaries that come with node-sqlite (solve https://github.com/mapbox/node-sqlite3/issues/1415) OR be able to install sqlite3@3.32 easily through `apt` on Ubuntu.
